snapcraft.yaml

```diff
@@ -49,6 +46,7 @@ parts:
     parse-info: [usr/share/metainfo/org.gnome.Mahjongg.appdata.xml]
     override-build: |
       sed -i -e 's|=org.gnome.Mahjongg$|=${SNAP}/meta/gui/org.gnome.Mahjongg.svg|g' $CRAFT_PART_SRC/data/org.gnome.Mahjongg.desktop.in
+      sed -i 's|<update_contact>mcatanzaro_at_gnome.org</update_contact>|<update_contact>mcatanzaro@gnome.org</update_contact>|g' $CRAFT_PART_SRC/data/org.gnome.Mahjongg.appdata.xml.in
```

Comment:
Mmm... maybe there is a reason for this... avoid spammer bots?

Reply:
removed that commit since the bug is fixed